my-app/src/Book.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import './Book.css';
+import defaultBook from './assets/default_book.svg';
+
+class Book extends Component {
```

Comment:
you have 2 types of components:
- dumb components: used to handle the view, they are declared as a function instead of a class because their only logic is the render function.

- smart components: they are declared as classes since they have multiple functions to handle the logic and their render function usually renders the dumb component.

---

this should be a dumb component